gh-100188: Reduce misses in BINARY_SUBSCR_LIST_INT (#100189)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2022-12-12-05-30-12.gh-issue-100188.sGCSMR.rst

> The ``BINARY_SUBSCR_LIST_INT`` instruction is no longer used for negative ``int``s, since that instruction always misses when encountering negative integers.

Author: ```suggestion
The ``BINARY_SUBSCR_LIST_INT`` instruction is no longer used for negative integers because that instruction always misses when encountering negative integers.
```

> The ``BINARY_SUBSCR_LIST_INT`` and ``BINARY_SUBSCR_TUPLE_INT``
> instruction are no longer used for negative integers because
> those instruction always miss when encountering negative integers.

Author: ```suggestion
The ``BINARY_SUBSCR_LIST_INT`` and ``BINARY_SUBSCR_TUPLE_INT``
instructions are no longer used for negative integers because
those instructions always miss when encountering negative integers.
```
